site/book/01-getting-started/02-quickstart.md

```diff
@@ -44,21 +44,33 @@ $ vim deployment.yaml
 Often, you want to automatically mutate and/or validate resources in a package.
 `kpt fn` commands enable you to execute programs called _kpt functions_.
 
-For example, you can automatically set a label with key `env` on all the resources in the package:
+For example, you can automatically search and replace the existing `port` value on all the resources in the package:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 -- env=dev
+$ kpt fn eval --image gcr.io/kpt-fn/search-replace:v0.1 -- by-value=80 put-value=8080
 ```
 
 `eval` command can be used for one-time _imperative_ operations. For operations that need to be
 performed repeatedly, there is a _declarative_ way to define a pipeline of functions as part of the
-package (in the `Kptfile`). This pipeline is executed using the `render` command:
+package (in the `Kptfile`).
+
+For example, you can declare `set-namespace` function and new desired value in the `pipeline` section of `Kptfile`:
```

Comment:
Explain what you want to accomplish, not just what to do. In this case, your intention is to set namespaces on all the resources.

Reply:
Added.